### PR TITLE
Example union job.

### LINKED ISSUE #755: An example job for Union component

### PR DESCRIPTION
**Pull request by tomaszguzialek**
Fixes #755

The most boring job ever, but from this data I have no better idea. Thought about "sending out an postcard / email with Christmas wishes to all the people connected to the organization, so customers and employees", but these two tables do not have uniform data... Anyone?
union-refresh-output-columns

### CODE REVIEW

**Comment by tomaszguzialek** [comment]
Version 2:
union2

**Comment by kaspersorensen** [comment]
It needs also to be copied into DC home and so on - like the other example jobs.

File: desktop/ui/src/main/resources/datacleaner-home/jobs/OrderDB Customers and Employees union.analysis.xml [hunk 1]
```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<job xmlns="http://eobjects.org/analyzerbeans/job/1.0">
+	<job-metadata>
```
Comment (kaspersorensen) [inline desktop/ui/src/main/resources/datacleaner-home/jobs/OrderDB Customers and Employees union.analysis.xml]: Let's strip the metadata for this type of job.

Comment (tomaszguzialek) [inline desktop/ui/src/main/resources/datacleaner-home/jobs/OrderDB Customers and Employees union.analysis.xml]: I removed the metadata about the author, timestamp etc., but left the coordinates of the components on the canvas.

Comment (kaspersorensen) [inline desktop/ui/src/main/resources/datacleaner-home/jobs/OrderDB Customers and Employees union.analysis.xml]: Good idea :)

**Comment by kaspersorensen** [comment]
LGTM. I suggest to merge when Travis is happy.

**Comment by ClaudiaPHI** [comment]
NIce job

**Pull request merged**